contacts_theme.theme

```diff
@@ -70,8 +70,31 @@ function contacts_theme_preprocess_block(&$variables) {
     ];
     foreach ($variables['content'] as $key => &$content) {
       if (in_array($key, $keys)) {
-        $content['button']['#value'] = '';
+        $content['#attributes']['class'] = array('dropdown');
+        $content['#prefix'] = '<li class="action-group-wrapper dropdown">';
+        $content['button']['#value'] = '+';
+        $content['button']['#attributes']['id'][] = 'dropdownMenuButton';
```

Comment:
We should probably use `\Drupal\Component\Utility\Html::getUniqueId` to avoid conflicts if there are multiple